### PR TITLE
Split the NoCode screen into separate views to simplify the logic

### PR DESCRIPTION
**Pull request by smcmurtry**
This will make it easier to add more logic to this screen in the future as we bring on more provinces and territories. This PR is purely a refactor - there should be no difference in the app (except for some bottom padding).

### CODE REVIEW

File: src/screens/nocode/NoCode.tsx [hunk 1]
```diff
@@ -1,140 +1,34 @@
...
-    </Box>
-  );
-};
-
-export const NoCodeScreen = () => {
-  const i18n = useI18n();
-  const navigation = useNavigation();
-  const close = useCallback(() => navigation.goBack(), [navigation]);
-  const onChooseRegion = useCallback(() => navigation.navigate('RegionSelect'), [navigation]);
+import {NoRegionView} from './views/NoRegionView';
```
Comment (timarney) [inline src/screens/nocode/NoCode.tsx:10]: These should be at the top of the file

**Comment by timarney** [comment]
Failing Detox test

Test one time code flow
    ✕ can get to the one time code screen (7341 ms)

  ● Test one time code flow › can get to the one time code screen

**Comment by timarney** [comment]
Looks like we're missing the external link icon

File: src/screens/nocode/NoCode.tsx (hunk 1 above)
Comment (smcmurtry) [reply src/screens/nocode/NoCode.tsx:10]: They are at the top - maybe it's all the deleted stuff in the diff making it look like they're farther down?

**Pull request merged**